README.md

```diff
@@ -31,69 +31,33 @@ In addition, there are some other features:
+1. **JuiceFS Client**: Coordinate the implementation of object storage and metadata storage engines, as well as file system interfaces such as POSIX, Hadoop, Kubernetes, and S3 gateway.
+2. **Data Storage**: Store the data itself, support local disk and object storage.
+3. **Metadata Engine**: Metadata corresponding to the stored data, supporting multiple engines such as Redis, MySQL, and SQLite;
 
-### Dependency
+![JuiceFS Architecture](docs/images/juicefs-arch-new.png)
 
-A Redis server (>= 2.8) is needed for metadata, please follow [Redis Quick Start](https://redis.io/topics/quickstart).
+JuiceFS relies on Redis to store file system metadata. Redis is a fast, open-source, in-memory key-value data store and very suitable for storing the metadata. All the data will store into object storage through JuiceFS client. [Learn more](docs/en/architecture.md)
 
-[macFUSE](https://osxfuse.github.io) is also needed for macOS.
+![JuiceFS Storage Format](docs/images/juicefs-storage-format-new.png)
 
-The last one you need is object storage. There are many options for object storage, local disk is the easiest one to get started.
+Any file stored in JuiceFS will be split into fixed-size **"Chunk"**, and the default upper limit is 64 MiB. Each Chunk is composed of one or more **"Slice"**. The length of the slice is not fixed, depending on the way the file is written. Each slice will be further split into fixed-size **"Block"**, which is 4 MiB by default. Finally, these blocks will be stored in the object storage. At the same time, JuiceFS will store each file and its Chunks, Slices, Blocks and other metadata information in metadata engines. [Learn more](docs/en/how_juicefs_store_files)
 
-### Format a volume
+![How JuiceFS stores your files](docs/images/how-juicefs-stores-files-new.png)
 
-Assume you have a Redis server running locally, we can create a volume called `test` using it to store metadata:
+Using JuiceFS, files will eventually be split into Chunks, Slices and Blocks and stored in object storage. Therefore, you will find that the source files stored in JuiceFS cannot be found in the file browser of the object storage platform. There is a chunks directory and a bunch of digitally numbered directories and files in the bucket. Don't panic, this is the secret of the high-performance operation of the JuiceFS file system!
 
-```bash
-$ ./juicefs format localhost test
-```
-
-It will create a volume with default settings. If there Redis server is not running locally, the address could be specified using URL, for example, `redis://username:password@host:6379/1`, the password can also be specified by environment variable `REDIS_PASSWORD` to hide it from command line options.
-
-**Note: After Redis 6.0.0, [`AUTH`](https://redis.io/commands/auth) command was extended with two arguments, i.e. username and password. If you use Redis < 6.0.0, just omit the username parameter in the URL, e.g. `redis://:password@host:6379/1`.**
```

Comment:
We need keep this information in somewhere.

Reply:
Maybe it's better to put these contents in "databases_for_metadata.md"? What do you think?

Reply:
Agreed